keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/EnvironmentPromotionCheckerTests.kt

```diff
@@ -277,7 +277,7 @@ internal class EnvironmentPromotionCheckerTests : JUnit5Minutests {
           /**
            * Verify that stateful constraints are not checked if a stateless constraint blocks promotion
            */
-          verify(exactly = 1) {
+          verify(inverse = true) {
```

Comment:
ohhhhh now I understand why it happened. thanks for fixing it!

Reply:
of course!